src/transformers/models/convnext/configuration_convnext.py

```diff
@@ -90,7 +90,6 @@ class ConvNextConfig(PretrainedConfig):
 
     def __init__(
         self,
-        num_channels=3,
```

Comment:
We can't remove `config.num_channels` since this will break the PyTorch modelling code! https://github.com/huggingface/transformers/blob/40082d598b756c4f1fb048c571ed95c31f05a69c/src/transformers/models/convnext/modeling_convnext.py#LL137C2-L137C2

Reply:
Hi, I should have mentioned the PR wasn't fully complete, I am aware of the error due to failed tests, I will fix it asap ( probably on this weekend )

Reply:
Cool! Sounds good! Feel free to ping me when it's ready and I'll get you a final review :)